docs/root/intro/version_history.rst

```diff
@@ -26,7 +26,7 @@ Version history
 * header to metadata: added :ref:`PROTOBUF_VALUE <envoy_api_enum_value_config.filter.http.header_to_metadata.v2.Config.ValueType.PROTOBUF_VALUE>` and :ref:`ValueEncode <envoy_api_enum_config.filter.http.header_to_metadata.v2.Config.ValueEncode>` to support protobuf Value and Base64 encoding.
 * http: added the ability to reject HTTP/1.1 requests with invalid HTTP header values, using the runtime feature `envoy.reloadable_features.strict_header_validation`.
 * http: added the ability to :ref:`merge adjacent slashes<envoy_api_field_config.filter.network.http_connection_manager.v2.HttpConnectionManager.merge_slashes>` in the path.
-* http: ignore h2c upgrades for HTTP/1.1 requests which are not currently supported.
+* http: remove h2c upgrade headers for HTTP/1 requests which are not currently supported.
```

Comment:
optional
which are not currently supported -> as h2c upgrades are not currently supported
just so it doesn't look like we're talking about HTTP/1 requests which are not currently supported.

Reply:
Thanx, cleared up.  Done.